controller/src/beerocks/cli/beerocks_cli.cpp

```diff
@@ -91,9 +91,18 @@ bool cli::parsedCommandArgumentsLinux(std::string line)
         resetArguments();
         return false;
     }
+
     const commandInfo &info = commandIt->second;
     commandNumOfArgs        = commandTokens.size() - 1;
 
+    int i                = 0;
```

Comment:
This declaration shouldn't have moved up with the rest, the variable is only used at the end.

Reply:
fixed